dl/src/main/scala/com/intel/analytics/bigdl/nn/TimeDistributedCriterion.scala

```diff
@@ -29,77 +29,66 @@ import scala.reflect.ClassTag
  * @param critrn
  */
 
-class TimeDistributedCriterion[T : ClassTag](critrn : TensorCriterion[T])
+class TimeDistributedCriterion[T : ClassTag](critrn : TensorCriterion[T],
+                                             sizeAverage: Boolean = true)
```

Comment:
The default value of sizeAverage in ClassNLLCriterion (or other criterions) is true, so I kept it as true. Is there any reason to change it as false?